File: src/API/SI.Domain/Enums/TransactionTypes.cs
﻿namespace SI.Domain.Enums;

public readonly struct TransactionTypes
{
    // <summary>
    // Nhập kho
    // </summary>
    public const string IMPORT = "IMPORT";

    // <summary>
    // Xuất kho
    // </summary>
    public const string EXPORT = "EXPORT";

    // <summary>
    // Điều chỉnh số lượng tồn kho
    // </summary>
    public const string ADJUST = "ADJUST";
}

File: src/FE-SmartInventory/src/Constant/TransactionTypes.ts
export const enum TransactionTypes {
    IMPORT = "IMPORT",
    EXPORT = "EXPORT",
    ADJUST = "ADJUST",
}
export const genCategoryEntityTypes= {
    [TransactionTypes.IMPORT]: 'Nhập kho',
    [TransactionTypes.EXPORT]: 'Xuất kho',
    [TransactionTypes.ADJUST]: 'Điều chỉnh số lượng tồn kho',
};